test/format/format.sh

```diff
@@ -18,14 +18,15 @@ trap 'onintr' 2
 
 usage() {
 	echo "usage: $0 [-aFSv] [-c config] "
-	echo "    [-h home] [-j parallel-jobs] [-n total-jobs] [-t minutes] [format-configuration]"
+	echo "    [-h home] [-j parallel-jobs] [-n total-jobs] [-p pre-setting] [-t minutes] [format-configuration]"
 	echo
 	echo "    -a           abort/recovery testing (defaults to off)"
 	echo "    -c config    format configuration file (defaults to CONFIG.stress)"
 	echo "    -F           quit on first failure (defaults to off)"
 	echo "    -h home      run directory (defaults to .)"
 	echo "    -j parallel  jobs to execute in parallel (defaults to 8)"
 	echo "    -n total     total jobs to execute (defaults to no limit)"
+	echo "    -p           pre settings to prepend before binary (defaults to empty)"
```

Comment:
"pre setting" doesn't really mean anything to me, I'd suggest something like "format-prefix"?

To be clear, I don't feel strongly, if you & @lukech prefer this, that's fbm.

Reply:
I agree. Updated the script to use "format-prefix"